src/run_terra_to_portal_data_post.py

```diff
@@ -0,0 +1,70 @@
+import igvf_and_terra_api_tools as api_tools
+import terra_to_portal_posting as terra2portal_transfer
+import argparse
+import logging
+import firecloud.api as fapi
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('--post_endpoint', type=str,
+                        help="""The POST endpoint, sandbox or production.""")
+    parser.add_argument('--terra_namespace', type=str,
+                        help="""Terra namespace name for the pipeline platform.""")
+    parser.add_argument('--terra_workspace', type=str,
+                        help="""Terra workspace name for the pipeline platform.""")
+    parser.add_argument('--terra_etype', type=str,
+                        help="""Terra eType name for the pipeline data table.""")
+    parser.add_argument('--upload_file', action='store_true',
+                        help="""If True, upload the file to the portal.""")
+    parser.add_argument('--output_dir', type=str,
+                        help="""Path to the output directory.""")
+    return parser
+
+
+def main():
+    parser = get_parser()
+    args = parser.parse_args()
+    terra_table = api_tools.get_terra_tsv_data_table(terra_namespace=args.terra_namespace,
+                                                     terra_workspace=args.terra_workspace,
+                                                     terra_etype=args.terra_etype)
+
+    # Get the API keys
+    igvf_api_keys = api_tools.set_up_api_keys(
+        igvf_endpoint=args.post_endpoint)
+
+    # Set up igvf utils connection
+    if args.post_endpoint == 'sandbox':
+        igvf_utils_mode = 'sandbox'
+    elif args.post_endpoint == 'production':
```

Comment:
is it possible that you use `prod` as well instead of `production` to keep consistent with igvf_utils